src/sensors/drivers/adafruit_dht22.cpp

```diff
@@ -123,9 +143,20 @@ void AdaDHT22::addCalibrationParametersToJSON(cJSON *json)
   cJSON_AddNumberToObject(json, CALIBRATION_TIME_STRING, configuration.cal_timestamp);
 }
 
-void AdaDHT22::configureDriverFromJSON(cJSON *json)
+bool AdaDHT22::configureDriverFromJSON(cJSON *json)
 {
-
+  const cJSON *sensorPinJSON = cJSON_GetObjectItemCaseSensitive(json, "sensor_pin");
+  int gpioPinCount = 7;  
+  if (sensorPinJSON->valueint > 0 && sensorPinJSON->valueint <= gpioPinCount)
+  {
```

Comment:
I kind of just like using the specific pin names, or port names with the understanding that certain ports include specific pins, but we don't have strict names for the ports yet either.

Reply:
Since you are the one using it now, let's go with whatever you think is going to be the most straightfoward for the user.  Making it all super clear does require board design/silkscreen changes.